backend/danswer/connectors/alation/connector.py

```diff
@@ -0,0 +1,228 @@
+import requests
+from collections.abc import Generator
+from datetime import datetime
+from enum import Enum
+from typing import Any
+from typing import Optional
+from urllib.parse import urlencode
+from urllib.parse import urlparse
+
+from bs4 import BeautifulSoup
+from danswer.configs.app_configs import INDEX_BATCH_SIZE
+from danswer.configs.constants import DocumentSource
+from danswer.configs.constants import HTML_SEPARATOR
+from danswer.connectors.interfaces import GenerateDocumentsOutput
+from danswer.connectors.interfaces import LoadConnector
+from danswer.connectors.models import Document
+from danswer.connectors.models import Section
+from danswer.utils.logging import setup_logger
+
+logger = setup_logger()
+
+
+class AlationObjectType(Enum):
+    """Object types in Alation. NOT an extensive list"""
+
+    ARTICLE = "article"
+    SCHEMA = "schema"
+    TABLE = "table"
+    COLUMN = "column"
+
+
+ALL_ALATION_OBJECT_TYPES = [
```

Comment:
I think ALL_ALATION_OBJECT_TYPES isn't needed

you can just later cast list(AlationObjectType) to get the same thing